tests/unit/test_schema.py

```diff
@@ -2049,6 +2049,46 @@ def test_add_meta_conditional_multicolumn_unique_constraint__ok():
         ]
 
 
+@pytest.mark.django_db
+@pytest.mark.skipif(django.VERSION[:2] < (5, 0), reason='functionality provided in django 5.0')
+@override_settings(ZERO_DOWNTIME_MIGRATIONS_RAISE_FOR_UNSAFE=True)
+def test_add_meta_unique_constraint_nulls_distinct__ok():
```

Comment:
I tried to write a test for the new `nulls_distinct` feature, but I must be missing something as it doesn't work. I'll try to spend more time on this later...

Reply:
Oh I think I misread the failure, I thought it was coming from the `assert editor.collected_sql` line, but I just realised that it came from the following one!